elements/bulbs-video/components/revealed.js

```diff
@@ -18,6 +18,13 @@ global.BULBS_ELEMENTS_COMSCORE_ID = '6036328';
 let jwPlayerIdCounter = 0;
 
 export default class Revealed extends React.Component {
+
+  componentWillUnmount () {
```

Comment:
Can we get a test that `componentWillMount` calls `this.player.stop`?

And don't need the extra blank lines.